### PR TITLE
[ricos-schema] map schema

### PR DESCRIPTION
**Pull request by maimonav**

### CODE REVIEW

File: packages/ricos-schema/web/src/generated/proto/wix/rich_content/v1/plugin_map.proto [hunk 1]
```diff
@@ -16,6 +16,13 @@ message MapData {
   MapSettings mapSettings = 2;
 }
 
+enum MapType {
+  ROADMAP = 0;
+  SATELINE = 1;
```
Comment (shaulgo) [inline packages/ricos-schema/web/src/generated/proto/wix/rich_content/v1/plugin_map.proto:21]:
```suggestion
  SATELLITE = 1;
```

Comment (maimonav) [reply packages/ricos-schema/web/src/generated/proto/wix/rich_content/v1/plugin_map.proto:21]: Thanks

File: packages/ricos-content/web/src/converters/draft/toDraft/convertDraftPluginData.ts [hunk 2]
```diff
@@ -281,7 +281,7 @@ const convertMapData = data => {
   data.mapSettings.isZoomControlShown = zoomControl;
   data.mapSettings.locationDisplayName = locationName;
   data.mapSettings.zoom = initialZoom;
-  data.mapSettings.mode = mapType;
+  data.mapSettings.mode = mapType?.lowerCase();
```
Comment (mharsat) [inline packages/ricos-content/web/src/converters/draft/toDraft/convertDraftPluginData.ts:284]: Typo - should be `toLowerCase`
Maybe you can add basic typing on `convertMapData`'s `data` object, it helped us in the past with cases like this

**Review by mharsat — CHANGES_REQUESTED** [review]
Can you add comments to the fields in the proto file? (like we have in `rich-content.proto`)

**Review by mharsat — APPROVED** [review]

**Pull request merged**